app/upgrades/v12/upgrades.go

```diff
@@ -46,6 +59,49 @@ func CreateV12UpgradeHandler(
 		}
 		keepers.FeeShareKeeper.SetParams(ctx, newFeeShareParams)
 
+		// ICA - https://github.com/CosmosContracts/juno/blob/integrate_ica_changes/app/app.go#L846-L885
+		vm[icatypes.ModuleName] = mm.Modules[icatypes.ModuleName].ConsensusVersion()
+
+		// create ICS27 Controller submodule params, controller module not enabled.
+		controllerParams := icacontrollertypes.Params{
+			ControllerEnabled: true,
+		}
+
+		// create ICS27 Host submodule params
+		hostParams := icahosttypes.Params{
+			HostEnabled: true,
+			AllowMessages: []string{
+				sdk.MsgTypeURL(&banktypes.MsgSend{}),
```

Comment:
this message list is outdated, perhaps this section should be removed as it was already included in v11 upgrade

Reply:
@giansalex We add more wasm messages in v12 than were in v11

Reply:
ok, this list should be updated to add v11 msgs (IbcTransfer, MsgUndelegate, etc)